test-projects/demo/dashboards/welcome.visivo.yml

```diff
@@ -71,9 +71,14 @@ dashboards:
           - width: 2
             table:
               name: Fibonacci Waterfall Table
-              trace: ref(Fibonacci Waterfall Trace)
+              traces:
+                - ref(Fibonacci Waterfall Trace)
               columns:
```

Comment:
We should make this optional and pull in all of the trace data if it's not specified.

Reply:
Perfect, same page.

Reply:
Included.